File: app/src/main/java/com/friday/ar/plugin/PluginViewHolder.java
package com.friday.ar.plugin;

import android.view.View;
import android.widget.ImageView;
import android.widget.TextView;

import androidx.annotation.NonNull;
import androidx.recyclerview.widget.RecyclerView;

import com.friday.ar.R;

public class PluginViewHolder extends RecyclerView.ViewHolder {
    TextView title;
    TextView path;
    ImageView icon;

    public PluginViewHolder(@NonNull View itemView) {
        super(itemView);
        title = itemView.findViewById(R.id.fileName);
        path = itemView.findViewById(R.id.fileSize);
        icon = itemView.findViewById(R.id.fileIcon);
    }
}

File: app/src/main/java/com/friday/ar/plugin/PluginViewHolder.kt
package com.friday.ar.plugin

import android.view.View
import android.widget.ImageView
import android.widget.LinearLayout
import android.widget.TextView
import androidx.recyclerview.widget.RecyclerView
import com.friday.ar.R

class PluginViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
    internal var root: LinearLayout = itemView.findViewById(R.id.itemRootLayout)
    internal var title: TextView = itemView.findViewById(R.id.fileName)
    internal var path: TextView = itemView.findViewById(R.id.fileSize)
    internal var icon: ImageView = itemView.findViewById(R.id.fileIcon)

}
